src/main/java/reposense/util/FileUtil.java

```diff
@@ -333,6 +334,15 @@ public static boolean isEmptyFile(String directoryPath, String relativeFilePath)
         return (new File(directoryPath, relativeFilePath).length() == 0);
     }
 
+    /**
+     * Returns the Bash expanded version of the {@code filePath}.
+     */
+    public static String getVariableExpandedFilePath(String filePath) {
+        String echoOutput = CommandRunner.runCommand(Paths.get("."), "echo " + filePath);
+        // CommandRunner returns some white space characters at the end
+        return StringsUtil.stripTrailingWhitespaces(echoOutput);
```

Comment:
Any reason why we need to keep the leading whitespaces? My thoughts about this is why are we not able to use `String.trim()` and we have to implement a custom function.

Reply:
Oops! I only knew about `String::strip` which is not available in Java 8. But it seems `trim()` works for this functionality. I have updated accordingly. Thanks for pointing out!